contracts/mocks/MockBosonRouter.sol

```diff
@@ -149,11 +149,11 @@ contract MockBosonRouter is
         nonReentrant
         whenNotPaused
     {
-        notAboveETHLimit(_metadata[2].mul(_metadata[5]));
-        notAboveETHLimit(_metadata[3].mul(_metadata[5]));
-        notAboveETHLimit(_metadata[4].mul(_metadata[5]));
-        require(_metadata[3].mul(_metadata[5]) == msg.value, "IF"); //invalid funds
-        //hex"54" FISSION.code(FISSION.Category.Finance, FISSION.Status.InsufficientFunds)
+        notAboveETHLimit(metadata[2].mul(metadata[5]));
+        notAboveETHLimit(metadata[3].mul(metadata[5]));
+        notAboveETHLimit(metadata[4].mul(metadata[5]));
+        require(metadata[3].mul(metadata[5]) == msg.value, "IF"); //invalid funds
+
 
```

Comment:
Some automatic merge probably went wrong here.
`_metadata` should be correct.
That's also why build is failing.

Same is true for next three changes in this file.

Reply:
I merged with main. This issue should be resolved.